app/src/main/java/fr/free/nrw/commons/depictions/models/Continue.java

```diff
@@ -0,0 +1,57 @@
+package fr.free.nrw.commons.depictions.models;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Model class for object obtained while parsing depiction response
+ */
+public class Continue {
```

Comment:
For the below classes ie. `Continue`, `Query`, `DepictionResponse` i believe that this could be part of `MwQueryResponse` and `MwQueryResult`. If yes, please avoid duplicating the classes.

Reply:
I think I am the least qualified person to assess this change as I have very little idea about the structure of the multitude of APIs used